project/Dependencies.scala

```diff
@@ -40,7 +40,7 @@ object Dependencies {
   val jodaTime = "joda-time" % "joda-time" % "2.9.9"
   val jodaConvert = "org.joda" % "joda-convert" % "1.8.3"
   val jSoup = "org.jsoup" % "jsoup" % "1.10.3"
-  val json4s = "org.json4s" %% "json4s-native" % "3.5.3"
+  val json4s = "org.json4s" %% "json4s-native" % "4.0.4"
```

Comment:
Update to json4s fixed issues with `Future` timeouts in tests:
<img width="1039" alt="image" src="https://user-images.githubusercontent.com/9575458/159023388-b727dfbb-b766-4941-83bf-a4401cbaa93a.png">
<img width="654" alt="image" src="https://user-images.githubusercontent.com/9575458/159023345-157f321b-94b6-4f52-8fe1-2c5c3b17cc6b.png">